libs/platform/src/lib/components/menu-button/menu-button.component.ts

```diff
@@ -1,21 +1,12 @@
-import {
-    Component,
-    Input,
-    Output,
-    EventEmitter,
-    ChangeDetectorRef,
-    ChangeDetectionStrategy,
-    ViewEncapsulation
-} from '@angular/core';
+import { Component, Input, Output, EventEmitter, ChangeDetectorRef, ChangeDetectionStrategy } from '@angular/core';
```

Comment:
do we need to change the formatting?

Reply:
removed unnecessary changes. but some formatting are done from prettier on saving the file. so have to keep those formatting.